config/sources/families/include/rockchip64_common.inc

```diff
@@ -197,6 +197,13 @@ prepare_boot_configuration() {
 		fi
 
 	fi
+
+	if [[ "$BOARD" == "rock-5a" ]];then
```

Comment:
When you declare `UBOOT_TARGET_MAP` in board config, it will get overrided in function `prepare_boot_configuration`. So I add this special board config at the end of this function.

Reply:
Oh, got it